.github/workflows/docs.yml

```
@@ -48,13 +48,16 @@ jobs:
 
       - name: pip install requirements
         run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
           python3 -m pip install -r ./scripts/discourse-sync/requirements.txt
```

If you want to avoid using `activate` you can do this:

```suggestion
          .venv/bin/python3 -m pip install -r ./scripts/discourse-sync/requirements.txt
```

Tbh, I don't know what the typical style is in a workflow, so I don't know which people would normally do (or something else entirely).